### PR TITLE
fix(cli): ensure we have correct values to replace .component-template with

### PR DESCRIPTION
**Pull request by jonnystening**
_ref_ https://github.com/WestpacGEL/GEL/issues/80#issuecomment-517086548

### CODE REVIEW

File: helpers/cli.js [hunk 1]
```diff
@@ -90,35 +90,40 @@ const main = async () => {
...
 
 	do {
 		description = await qDescription().catch(e => console.log(chalk.red(e)));
 	} while (!description);
 
-	const folderName = name.toLowerCase();
+	const folderName = packageName.toLowerCase();
 
-	name = name
+	const compomentTitle = packageName
```
Comment (jaortiz) [inline helpers/cli.js]: You misspelt component :joy:

File: helpers/cli.js [hunk 2]
```diff
@@ -90,35 +90,40 @@ const main = async () => {
...
+		.split('-')
+		.map(part => part[0].toUpperCase() + part.slice(1))
+		.join(' ');
+
+	const componentName = packageName
```
Comment (jaortiz) [inline helpers/cli.js]: we can just do 
```
const componentName = componentTitle;
```
one less calculation

Comment (jonnystening) [reply helpers/cli.js]: componentName has no spaces ("FormPod"), componentTitle has spaces ("Form Pod")

Comment (jonnystening) [reply helpers/cli.js]: how about

```const componentName = componentTitle.replace(' ', '');```

File: helpers/cli.js (hunk 1 above)
Comment (jonnystening) [reply helpers/cli.js]: 😳

**Pull request merged**